src/common/types.ts

```diff
@@ -168,6 +168,7 @@ export interface SettingsData {
   startupMode?: StartupMode;
   zoomLevel?: number;
   notificationsEnabled?: boolean;
+  darkMode?: boolean;
```

Comment:
Let's have this as `theme: 'dark' | 'light'` instead of a boolean flag, so we can support more themes in the future.